server/api/index.js

```diff
@@ -4,18 +4,18 @@ const router = express.Router();
 
 //add additional routes here
 router.use('/order', require('./order'));
-router.use('/orderitem', require('./orderitem'));
-router.use('/user', require('./user'));
+router.use('/orderitems', require('./orderitem'));
+router.use('/user', require('./user'))
 router.use('/cartitem', require('./cartitem'))
 router.use('/cart', require('./cart'))
 
 router.use('/products', require('./products'))
 
 //error handling
 router.use((req, res, next) => {
-  const err = new Error(`Invalid API path: ${req.originalUrl}`);
+  const err = new Error(`Invalid API path: ${req}`);
```

Comment:
Just curious why you removed this?
If you console log req, it appears as just [Object object]
By logging the req.originalUrl, we can see what the actual path the request was sent on, which didn't match something in this index

Reply:
Wasn't intentional. Not sure what happened, but we can definitely put the originalUrl back in.

Working on merge conflicts now and will submit another PR when I have that sorted.